docs/release_process.md

```diff
@@ -14,6 +14,15 @@ The release process should result in the following side effects:
 
 Note all release tags pushed to GitHub MUST follow semantic versioning.
 
+## Prerequisites
+
+Before attempting releases of `cellxgene-schema` to Test PyPI and PyPI, create an account on each:
+* https://pypi.org/account/register/
+* https://test.pypi.org/account/register/
+
+and request to be added to the `Owner` role in the [#single-cell-eng](https://czi-sci.slack.com/archives/C023Q1APASK) Slack channel.
```

Comment:
`Owner` role for which account? Does CZI have a single group account for PyPI and Test PyPI...?

Reply:
ah, I see... it's for the package